kbfsfuse/main.go

```diff
@@ -80,6 +80,13 @@ func start() *libfs.Error {
 	} else {
 		mountDir = flag.Arg(0)
 	}
+	dir, err := os.Stat(mountDir)
+	if err != nil || !dir.IsDir() {
```

Comment:
You can check for a more specific `err` here by using `os.IsNotExist(err)`, instead of `err != nil`.  Presumably if you get some other error (say the disk is completely borked), you'd want to return a `libfs.InitError` as above, rather than trying to `mkdir`.